### PR TITLE
hashbang.html: More direct one-liner

### PR DESCRIPTION
**Pull request by KellerFuchs**
Also explains in the comments why piping the output from GnuPG is dangerous.

### CODE REVIEW

File: src/hashbang.html [hunk 1]
```diff
@@ -61,7 +69,7 @@
     <script src='/assets/icon.js'></script>
     <h1>#!</h1>
     <a href="view-source:https://hashbang.sh">
-      <code>curl hashbang.sh | gpg > hashbang.sh</code>
+      <code>curl hashbang.sh | gpg | sh</code>
```
Comment (stefan2904) [inline src/hashbang.html]: if I manually add a https here things are getting weird

Comment (ChickenNuggers) [inline src/hashbang.html]: Half the point of the message is to indicate that you -shouldn't- run it using a oneliner; `curl hashbang.sh | gpg > hashbang.sh; vim hashbang.sh` then when you're satisfied with the file, `sh hashbang.sh`.

Comment (ChickenNuggers) [inline src/hashbang.html]: I think the issue might be with pumping GPG (or any output containing the command `read`) into sh where sh starts reading. You'd have to do something like `sh <(curl https://hashbang.sh | gpg)` but, again, DON'T do this.

**Comment by daurnimator** [comment]
The command now line wraps. Please fix this. 

With this merged (which I don't necessarily like), we have no reason to insist on https any more.

**Comment by lrvick** [comment]
https gives one extra point of assurance that we are not telling lies about our key id.

Nothing on the internet should allow http anymore :-P

**Comment by KellerFuchs** [comment]
@daurnimator It can be reverted.  The main intent, though, was to indeed send people to `warn.sh`, to give them a reminder of the dangers of `curl | sh`.

**Pull request merged**